.github/workflows/iroha2-dev-pr.yml

```diff
@@ -128,7 +128,7 @@ jobs:
       - name: Install latest rust
         uses: actions-rs/toolchain@v1
         with:
-          toolchain: stable
+          toolchain: 1.52
```

Comment:
Can we use 1.52.1?
As 1.52 sometimes produced faulty binaries :(
https://blog.rust-lang.org/2021/05/10/Rust-1.52.1.html

Reply:
This will install the latest 1.52.x

Reply:
Which means that all the hotfixes will be available instantly, but the breaking changes will need to be migrated manually.

Reply:
Ah, okay then!